cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -90,4 +90,21 @@
     </li>
   </ol>
 </section>
+
```

Comment:
Update class name below "details" to be consistent with the actual HTML. Actually, make sure this whole block of mock HTML (used for tests) is consistent with the actual HTML.